src/Magicodes.ExporterAndImporter.Csv/CsvExporter.cs

```diff
@@ -87,5 +87,35 @@ public Task<byte[]> ExportHeaderAsByteArray<T>(T type) where T : class, new()
             var helper = new ExportHelper<T>();
             return Task.FromResult(helper.GetCsvExportHeaderAsByteArray());
         }
+
+        public Task<byte[]> ExportHeaderWithXSSFWorkbookAsByteArray<T>(T type) where T : class, new()
```

Comment:
It looks like in the interface, CsvExporter have been affected, maybe we need to do something about it in the near future.

Reply:
I can remove it, should I ?